### DISCUSSION TITLE
NO ALERTS IN WEB INTERFACE (Standalone architecture)

### DISCUSSION THREAD

**Discussion by Amelie80** [Security-Onion-Solutions/securityonion#14011] [Unsupported Versions]
Bonjour, J'ai déployé Security Onion depuis déjà  3 mois environs et tout fonctionnait bien. Depuis maintenant une semaine, j'ai remarque que les alertes ne remontaient pas (pas d'alertes dans l'onglet alerts)
![image](https://github.com/user-attachments/assets/2bb045da-4fc1-404d-afa3-5233132af084)

![image](https://github.com/user-attachments/assets/57f52cf4-350b-4d91-8897-b650ec57f11b)

J'ai essayé de résoudre ce problème en me basant sur les recommandations qui on déjà été fourni dans certaines discussion qui traitaient plus ou moins la même problématique (mise à jour des règles )  mais cela n'a pas été favorable;
![image](https://github.com/user-attachments/assets/66ce11b6-843d-420a-9c10-c23205e52c03)
![image](https://github.com/user-attachments/assets/d83fb3b6-af44-4d11-a9bd-8589958af2da)

j'ai donc, effectuer un so-test : 
sauf qu'ici les alertes s'affichent, mais les alertes qui sont remontées ne concernent pas du tout mon réseau , c'est des plages d'adresse IP qui ne font pas du tout partis de mon réseau à surveiller . Avez-vous une idée ou une suggestion sur la manière de résoudre ce problème d'alertes? Aussi j'aimerais mentionner que certes les fausse alertes s'affiche (avec les mauvaise plages d'adresse), mais le lendemain lorsque je reviens tout redeviens comme avant (pas d'alertes carrément). et je refais à chaque fois le so-test …Merci d'avance pour votre assistance . 

Architecture : standalone
RAM : 64 Go
disk : 250 Go

**Comment by rh-ops** [comment]
Lets check a few things. First your storage. What is the result of the following command? `df -h /nsm`. 

Then we can check your cluster health with the next command `sudo so-elasticsearch-query _cluster/health?pretty`

If the result shows the cluster status as yellow or red, run the following command `sudo so-elasticsearch-query _cluster/allocation/explain?pretty` and reply with the result.

**Reply by Amelie80** [reply]
Voici les sorties :

~]$ df -h /nsm
Filesystem              Size  Used Avail Use% Mounted on
/dev/mapper/system-nsm  151G   70G   82G  47% /nsm

pour la requete : sudo so-elasticsearch-query _cluster/health?pretty
  "cluster_name" : "securityonion",
  "status" : "green",
  "timed_out" : false,
  "number_of_nodes" : 1,
  "number_of_data_nodes" : 1,
  "active_primary_shards" : 191,
  "active_shards" : 191,
  "relocating_shards" : 0,
  "initializing_shards" : 0,
  "unassigned_shards" : 0,
  "delayed_unassigned_shards" : 0,
  "number_of_pending_tasks" : 0,
  "number_of_in_flight_fetch" : 0,
  "task_max_waiting_in_queue_millis" : 0,
  "active_shards_percent_as_number" : 100.0
}

**Reply by rh-ops** [reply]
Alright all that looks good. This may be a networking issue then. Lets run a tcp dump on your bond0 interface with the following command `tcpdump -nni bond0`. Depending on if you see any layer 7 or layer 2 traffic will tell us where the issue is.

**Comment by Amelie80** [comment]
d'après la sortie de tcpdump -nni bond0, il s'agit d'un trafic de couche 2 : 
Voici une pétite capture : 
![image](https://github.com/user-attachments/assets/395d1317-0d2c-4d0e-9b1e-558e1d606dbd)

**Reply by rh-ops** [reply]
Alright so in that case I'd recommend checking all your physical connections.

**Comment by Amelie80** [comment]
Merci pour votre assistance. Le problème était effectivement au niveau 2. Tout est à présent fonctionnel.
